package.json

```diff
@@ -18,13 +18,17 @@
   "devDependencies": {
     "babel-eslint": "^4.1.5",
     "eslint": "^1.10.1",
-    "eslint-config-atom-build": "^1.0.0"
+    "eslint-config-atom-build": "^1.0.0",
+    "atom-build-spec-helpers": "^0.4.0",
+    "temp": "^0.8.3",
+    "fs-extra": "^0.23.1"
   },
   "keywords": [
     "build",
     "compile",
     "elixir",
     "productivity"
   ],
-  "main": "lib/elixir.js"
+  "main": "lib/elixir.js",
+  "scripts": { "test": "atom --test spec/*" }
```

Comment:
This package has a `.travis.yml` but never really run or used.

I'm happy to merge this without travis support, but if you wanna add it, the instructions on how to do so can [be found here](https://github.com/atom/ci#travis-ci).